core/plugin.php

```diff
@@ -85,7 +85,8 @@ public function save($plugin_options_form) {
 			$result = DB::exec("update plugins set options=? where id=?", [$options_json, $this->id]);
 			
 			if ($result) {
-				CMS::Instance()->queue_message('Plugin options updated','success',Config::uripath() . '/admin/plugins/show');	
+				$msg = 'Plugin <a href="' . Config::uripath() . '/admin/plugins/edit/' . $this->id . '">' . $this->title . '</a> ' . 'updated';
```

Comment:
templating?

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09